cdap-app-fabric-tests/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java

```diff
@@ -431,7 +434,27 @@ public void testWorkflowPauseResume() throws Exception {
     suspendWorkflow(programId, runId, 404);
 
     resumeWorkflow(programId, runId, 404);
+  }
+
+  @Category(XSlowTests.class)
+  @Test
+
+  public void testKillSuspendedWorkflow() throws Exception {
+    HttpResponse response = deploy(SleepingWorkflowApp.class, Constants.Gateway.API_VERSION_3_TOKEN, TEST_NAMESPACE2);
+    Assert.assertEquals(200, response.getStatusLine().getStatusCode());
+
+    WorkflowId workflow = Ids.namespace(TEST_NAMESPACE2).app("SleepWorkflowApp").workflow("SleepWorkflow");
+
+    // Start the workflow
+    startProgram(workflow.toId(), 200);
```

Comment:
Remove all calls of `.toId()` . This method converts `WorkflowId` to a deprecated class